runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -824,6 +835,16 @@ TR_J9EstimateCodeSize::processBytecodeAndGenerateCFG(TR_CallTarget *calltarget,
             TR::Node *parent = 0;
             TR::Node *callNode = 0;
             TR::ResolvedMethodSymbol *resolvedSymbol = 0;
+
+            if (resolvedMethod)
+               {
+               const char * sig = resolvedMethod->signature(comp()->trMemory());
+               if (sig && !strncmp(sig, "java/util/HashMap.hash", 22))
```

Comment:
Similar question to before for this spot too. Should/could we be using recognized methods here ? Please check the other `strncmp` uses that you added as well. At least one of the other ones is only known to be a "method" rather than a "resolved method". So you may not be able to use recognized methods there.